arangod/RocksDBEngine/RocksDBOptionFeature.cpp

```diff
@@ -1276,12 +1504,29 @@ void RocksDBOptionFeature::validateOptions(
 }
 
 void RocksDBOptionFeature::prepare() {
+  if (_enableBlobFiles) {
+    LOG_TOPIC("5e48f", WARN, Logger::ENGINES)
+        << "using blob files is experimental and not supported for production "
+           "usage";
+  }
+
   if (_compactionStyle != ::kCompactionStyleLevel) {
     LOG_TOPIC("6db54", WARN, Logger::ENGINES)
         << "using compaction style '" << _compactionStyle
         << "' is experimental and not supported for production usage";
   }
 
+  if (_blockCacheType == ::kBlockCacheTypeHyperClock) {
+#ifndef ARANGODB_ROCKSDB8
+    // cannot be reached with RocksDB 7.2
+    TRI_ASSERT(false);
```

Comment:
Should we make this a ADB_PROD_ASSERT? I know we should never be able get here, but let's play safe. :)